bio/mmseqs2/workflows/wrapper.py

```
@@ -0,0 +1,37 @@
+from snakemake_wrapper_utils.snakemake import get_format
+
+extra = snakemake.params.get("extra", "")
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+
+target = snakemake.input.get("target", "")
+if isinstance(target, list):
+    target = os.path.commonprefix(snakemake.input.target)
+
+# TODO: arbitrary output file names
+out = snakemake.output
```

Ruff (0.13.1)

Undefined name `snakemake`

(F821)